src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height;
+    private int topBase;
+    private int botBase;
+
+    public IsoscelesTrapezoid(String color, int height, int topBase, int botBase) {
+        super(color);
+        this.height = height;
+        this.topBase = topBase;
+        this.botBase = botBase;
+    }
+
+    public IsoscelesTrapezoid(String color) {
+        super(color);
+    }

```

Comment:
Do you use this construct? if not, let’s  remove it

Reply:
not fixed